charts/opentelemetry-collector/values.schema.json

```diff
@@ -493,7 +493,7 @@
     },
     "service": {
       "type": "object",
-      "additionalProperties": false,
+      "additionalProperties": true,
```

Comment:
Instead of setting this to true please add "None" to the type Enum list below.

Reply:
fixed, thanks.